fireworks_poe_bot/__main__.py

```diff
@@ -41,7 +41,7 @@ def main(args=None):
         parser = argparse.ArgumentParser(
             prog="fireworks_poe_bot",
             description=f"""
-        Fireworks LLM Poe Server Bot v0.0.1.
+        Fireworks LLM Poe Server Bot
```

Comment:
`image-size` is actually unused, I'll delete it. The others are server-related args and make sense to take as arguments to the server rather than in config